app/assets/javascripts/hyrax/autocomplete.es6

```diff
@@ -1,44 +1,37 @@
-import Default from './autocomplete/default';
-import Work from './autocomplete/work';
+import Default from './autocomplete/default'
+import Work from './autocomplete/work'
 
 export default class Autocomplete {
-    constructor(options) {
-	this.autocompleteFields = options.autocompleteFields;
-    }
-    // This is the initial setup for the form.
-    setup() {
-	$('[data-autocomplete]').each((index, value) => {
-            let selector = $(value);
-	    let autocompleteData = selector.data('autocomplete');
-	    this.activateFields(autocompleteData,selector);
-	});
-    }
-    // This activates autocomplete for added fields
-    fieldAdded(cloneElem) {
-	let selector = $(cloneElem);
-	let autocompleteData = selector.data('autocomplete');
-	this.activateFields(autocompleteData,selector);
-    }
-    autocomplete(field) {
-	let fieldName = field.data('autocomplete');
-	switch (fieldName) {
-	case "work":
-	    new Work(
-              field,
-              field.data('autocomplete-url'),
-              field.data('exclude-work')
-	    );
-	    break;
-	default:
-	    new Default(field, field.data('autocomplete-url'));
-	    break;
-	}
-    }
-    activateFields(autocompleteData, selector) {
-	for (let field in this.autocompleteFields) {
-	    if (autocompleteData === this.autocompleteFields[field]) {
-		this.autocomplete(selector);
-	    }
-	}
+  // This is the initial setup for the form.
+  setup () {
+    $('[data-autocomplete]').each((index, value) => {
```

Comment:
Can we move this initialization code to the `app.js`. I'd like to pass the selected object to the initializer, then this class doesn't need to know about `$`. Ideally it would pass the result of `.data()` too.